ui/DesignSystem/Primitive/Input/Validated.svelte

```diff
@@ -0,0 +1,112 @@
+<script>
```

Comment:
`Input/Text.svelte` already implemented this variant. Maybe making a new component out of it is clearer, but the naming `Validated` is confusing, as all inputs should be validated and the variant should then be removed from `Input/Text.svelte`

Reply:
Makes sense! I updated `Input/Text`